pkg/build/builder/sti.go

```diff
@@ -226,12 +228,14 @@ func (s *S2IBuilder) Build() error {
 
 	// If DockerCfgPath is provided in buildapiv1.Config, then attempt to read the
 	// dockercfg file and get the authentication for pulling the images.
+	//TODO in theory change to GetAllDockerAuths once PullAuthentication can take multiples
 	t, _ := dockercfg.NewHelper().GetDockerAuth(config.BuilderImage, dockercfg.PullAuthType)
 	config.PullAuthentication = s2iapi.AuthConfig{Username: t.Username, Password: t.Password, Email: t.Email, ServerAddress: t.ServerAddress}
```

Comment:
so yeah, this field from the openshift/source-to-image repo only takes a single registry auth at the moment

Reply:
the `result, err := builder.Build(config)` call further down in this file ultimately consumes it

Reply:
or wait, do we even care here ? .... s2i is suppose to be the asDockerfile in 4.x .... investigating

Reply:
Yeah the auths are gathered from the `pullAuthConfigs, err := s.setupPullSecret()` call further down in this method

And the dockerfile builder in s2i does not make any RPCs over the docker client

I'd contend that these 2 lines can be deleted.  

I may do that in a separate commit and see what the CI results show us.

Reply:
with the responses to @mtrmac comments I believe this has become a no-op